WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/InPersonPaymentsMenuViewModel.swift

```diff
@@ -82,36 +85,38 @@ class InPersonPaymentsMenuViewModel: ObservableObject {
             .assign(to: &$shouldBadgeTapToPayOnIPhone)
 
         Task { @MainActor in
-            await updateOutputProperties(trackAnalytics: false)
+            _ = try? await dependencies.systemStatusService.synchronizeSystemInformation(siteID: siteID)
+            await updateOutputProperties()
```

Comment:
Should we change the optional `try` with a do-catch block here?
```suggestion
            do {
                _ = try await dependencies.systemStatusService.synchronizeSystemInformation(siteID: siteID)
                await updateOutputProperties()
            } catch {
                DDLogError("Oopsie: \(error)")
            }
```

Reply:
Hmm. I'm not sure how much we care if the sync fails, it doesn't feel important enough to log, and it adds a fair bit of visual clutter to the function.

The subsequent plugin fetch will still work, using stored data, and other parts of the app update the system info pretty regularly too.